bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -0,0 +1,88 @@
+package com.google;
+
+import static com.google.sps.data.Keys.ID_ENTITY_PROPERTY;
+import static com.google.sps.data.Keys.USER_ENTITY;
```

Comment:
It looks like these imports are unused.  I'd delete them and bring them back later when you're actually ready to use them.

Reply:
Removed